src/App.jsx

```diff
@@ -14,33 +20,78 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const getSortedGoods = (goods, sortBy, isReversed) => {
+  const sortedGoods = [...goods];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  if (sortBy === SORT_FIELD_ALPHABET) {
+    sortedGoods.sort((a, b) => a.localeCompare(b));
+  } else if (sortBy === SORT_FIELD_LENGTH) {
+    sortedGoods.sort((a, b) => a.length - b.length);
+  }
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (isReversed) {
+    sortedGoods.reverse();
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  return sortedGoods;
+};
+
+export const App = () => {
+  const [sortBy, setSortBy] = useState('');
```

Comment:
The initial state of `sortBy` is set to an empty string. If the application requires a default sorting order, consider setting `sortBy` to either `SORT_FIELD_ALPHABET` or `SORT_FIELD_LENGTH` to ensure consistent behavior.